packaging/windows/smtk2ssrf-mxe-build.sh

```diff
@@ -222,6 +222,7 @@ i686-w64-mingw32.shared-cmake \
 	-DPKG_CONFIG_EXECUTABLE="/usr/bin/pkg-config" \
 	-DCMAKE_PREFIX_PATH="$BASEDIR"/mxe/usr/i686-w64-mingw32.shared/qt5 \
 	-DCMAKE_BUILD_TYPE=$RELEASE \
+	-DNO_SSRF_QML=ON \
```

Comment:
you can drop this commit now - I simply added the missing link libraries which seemed much easier and less intrusive.

Reply:
ok will do that tomorrow.